prow/repoowners/repoowners.go

```diff
@@ -217,12 +220,16 @@ func (c *Client) LoadRepoOwners(org, repo, base string) (*RepoOwners, error) {
 	var owners *RepoOwners
 	// Filter collaborators. We must filter the RepoOwners struct even if it came from the cache
 	// because the list of collaborators could have changed without the git SHA changing.
-	collaborators, err := c.ghc.ListCollaborators(org, repo)
-	if err != nil {
-		log.WithError(err).Errorf("Failed to list collaborators while loading RepoOwners. Skipping collaborator filtering.")
-		owners = entry.owners
+	if !c.skipCollaborators(org, repo) {
```

Comment:
nit: I'd invert this and avoid the indentation:
```go
if c.skipColl() {
  return entry.owners, nil
}
// previous code
```

Reply:
done